Sming/compiler/ld/eagle.app.v6.cpp.include.ld

```diff
@@ -0,0 +1,217 @@
+/*  
```

Comment:
Hi @anakod, I think you are right. I was not able to reproduce the problem, but it looked exactly like the screenshot you provided in #946.